modules/tools/mapshow/mapshow.py

```diff
@@ -98,5 +102,9 @@ def draw(map):
         localization.load(args.loc)
         localization.plot_vehicle(plt)
 
+    if args.position is not None:
+        x = float(args.position.split(",")[0])
+        y = float(args.position.split(",")[1])
```

Comment:
Split once and reuse. Eg:
`x, y = args.position.split(",")`
`x, y = float(x), float(y)`

Reply:
Thanks for pointing this out! Changing it now.